examples/setup-fee-distributor-contract/index.ts

```diff
@@ -0,0 +1,175 @@
+import {
+  createPublicClient,
+  createWalletClient,
+  http,
+  defineChain,
+  parseAbi,
+  getAddress,
+} from 'viem';
+import { generatePrivateKey, privateKeyToAccount } from 'viem/accounts';
+import { arbitrumSepolia } from 'viem/chains';
+import {
+  feeRouterDeployRewardDistributor,
+  createRollupFetchCoreContracts,
+  createTokenBridgeFetchTokenBridgeContracts,
+  arbOwnerPublicActions,
+} from '@arbitrum/orbit-sdk';
+import { sanitizePrivateKey } from '@arbitrum/orbit-sdk/utils';
+import { config } from 'dotenv';
+config();
+
+// environent variables check
+if (typeof process.env.ROLLUP_ADDRESS === 'undefined') {
+  throw new Error(`Please provide the "ROLLUP_ADDRESS" environment variable`);
+}
+
+if (typeof process.env.CHAIN_OWNER_PRIVATE_KEY === 'undefined') {
+  throw new Error(`Please provide the "CHAIN_OWNER_PRIVATE_KEY" environment variable`);
+}
+
+if (typeof process.env.ORBIT_CHAIN_ID === 'undefined') {
+  throw new Error(`Please provide the "ORBIT_CHAIN_ID" environment variable`);
+}
+
+if (typeof process.env.ORBIT_CHAIN_RPC === 'undefined') {
+  throw new Error(`Please provide the "ORBIT_CHAIN_RPC" environment variable`);
+}
+
+// load the chain owner account (or any account that has the executor role in the UpgradeExecutor)
+const chainOwner = privateKeyToAccount(sanitizePrivateKey(process.env.CHAIN_OWNER_PRIVATE_KEY));
+
+// creating some random addresses to show the example
+const randomAccount = privateKeyToAccount(generatePrivateKey());
```

Comment:
I guess I'm unclear on wether or not these scripts are intended to be run against mainnet (if so, these should be provided as inputs, maybe as a json file)

Reply:
These scripts are meant to be examples only. They can be run on mainnet, but should be configured first (even if you run it on testnet, you'll have to configure a few things). Random data is used sometimes like for the batch poster in the createRollup example.

Reply:
I tried to provide a description of how to configure the different recipient addresses and their weights though.

Reply:
Examples were modified so all inputs are now provided in the .env file